utils/get-project-content.js

```diff
@@ -107,6 +107,13 @@ function _getCollectionFragments(collectionDirectory) {
           }
         };
 
+        if (metadata.thumbnailPath) {
+          metadata.thumbnailPath = path.resolve(
```

Comment:
We shouldn't put the full thumbnail path in the project content. If we need this, it should be relative to the project base (ex. `src/collection/fragment/thumbnail.jpg`